### DISCUSSION TITLE
Markdown formatting with inline code spans using backticks

### DISCUSSION THREAD

**Discussion by segevfiner** [mermaid-js/mermaid#4575] [Q&A]
This doesn't currently work:
```
flowchart TD
    A --> B["`This is a `foobar``"]
```

Is there any syntax to be able to use backticks inside Markdown formatted text in the flowchart diagram type?

**Comment by knsv** [comment]
😱 

Sorry ... not without some changes

**Comment by sidharthv96** [comment]
We only support _italics_ and **bold** for now.

**Comment by roke-julian-lockhart** [comment]
> [`#discussion-5360148`][1]

[1]: https://github.com/orgs/mermaid-js/discussions/4575#discussion-5360148

@segevfiner, I expect that this would break [`stackoverflow.com/revisions/75978908/2`][2] (see https://github.com/mermaid-js/mermaid/issues/5319#issue-2153013695).

[2]: https://stackoverflow.com/revisions/75978908/2

**Comment by Piratepie** [comment]
There is a way to use backticks inside of markdown. The only way to do so is to use [entity codes](https://mermaid.js.org/syntax/flowchart.html#entity-codes-to-escape-characters).

The drawback, it'll render as just a backtick, so no pretty code markdown like `this`.

Here's me updating your example:

```
flowchart TD
    A --> B["`This is a #96;foobar#96;`"]
```

Which will look like this in the mermaid editor (doesn't seem to work on GitHub)
![image](https://github.com/mermaid-js/mermaid/assets/27888345/c30ed1ba-598d-46f7-a2a7-495d4da2e3d6)

**Reply by roke-julian-lockhart** [reply]
> [`#discussioncomment-8593317`][1]

[1]: https://github.com/orgs/mermaid-js/discussions/4575#discussioncomment-8593317

Yeah, @Piratepie, that's gotta be, since Mermaid doesn't allow strings to be delimited by <code>\`\`</code> instead of <code>\`</code>. If it did, it would permit the standard workaround of [`meta.stackexchange.com/revisions/82722/8`][2] to work.

[2]: https://meta.stackexchange.com/revisions/82722/8

**Comment by phil294** [comment]
You can already write `<code>...</code>`, this works well - unless you write `https://` *in* the inline code, then you'll get `Unsupported markdown: link`, in which case you can use `:` instead of `:`. The former is `U+A789 ꞉ MODIFIER LETTER COLON`, the latter `U+003A : COLON`.

**Reply by roke-julian-lockhart** [reply]
@phil294, can a semantic `COLON` not be escaped? Needing to use unsemantic replacements that, may not appear equivalent based upon typeface, is not desirable, for the sake of screen readers.

**Reply by phil294** [reply]
I don't know, you tell me. I'm not a mermaid contributor.

**Reply by roke-julian-lockhart** [reply]
@phil294, nor am I. If I knew, I wouldn't need to ask. 😔
